README.md

```diff
@@ -119,7 +119,24 @@ Ensure you also create the `procbots/versionbot/ready-to-merge` label type in th
 
 Finally you need to install the Integration into the repo. Do this by going to your 'Settings' page, selecting 'Installed Integrations', selecting your Integration and then selecting the repos you want it installed in in the 'Repository access' section.
 
-## Running
+### Tailoring ProcBots for a Repo via Configuration File
+
+ProcBots now also respond to a configuration file. This is a file with the name `.procbots.yml` in a relative location for the ProcBot running. In the case of the VersionBot, this is in the root of the repository that it is working on.
```

Comment:
Should this specify "In the case of the VersionBot, this is in the root of the repository that it is working on, on the `master` branch." ?  (just for removal of any doubt)